docs/reference/modules/page.md

```diff
@@ -0,0 +1,379 @@
+|`targetId` | String | The `_id` of an existing page to use as a target when inserting the new page. `_home` and `_archive` are optional conveniences for the home page and [archived section](/reference/api/pages.md#moving-pages-to-the-archive), respectively. |
+|`position` | Integer/String | A numeric value will represent the zero-based child index under the `_targetId` page. `before`, `after`, `firstChild`, or `lastChild` values set the position within the page tree for the new page in relation to the target page (see `_targetId`). `before` and `after` insert the new page as a sibling of the target. `firstChild` and `lastChild` insert the new page as a child of the target. |
+
+### `async archive(req, _id)`
+
+The `archive()` method moves a page, identified by its unique `_id`, into the page tree's archive section. It returns and object with two properties: `parentSlug`, the slug of the page's former parent; and `changed`, an array of objects with `_id` and `slug` properties, identifying all child pages of the moved page that were also archived.
+
+### `async publish(req, draft, options)`
+
+When passed a `req` object and *draft* document object (`draft`), this method will publish the draft. This replaces an existing published version of the page, if there is one. The options object (`options`) is currently only used for internal draft state management.
+
+### `async localize(req, draft, locale, option)`
+
+Localize the draft page (`draft`), copying it to another locale (`locale`). This creates that locale's draft for the first time if necessary. By default existing documents are not updated, but setting `update: true` in the `options` object will update existing ones.
+
+### `async revertDraftToPublished(req, draft)`
+
+Reverts the given draft page (`draft`) to the most recent publication, clearing any changes. It returns the draft's new value, or `false` if the draft was not modified from the published version or no published version exists yet.
+
+Emits the [`afterRevertDraftToPublished` event](/docs/reference/server-events.md#afterrevertdrafttopublished) before returning, which includes the draft document in its payload, allowing you to alter the returned draft object.
+
+### `async revertPublishedToPrevious(req, published)`
+
+Reverts a published page document (`published`) to the previous published state and returns the updated published state. If this was already done (only one previous state is saved) or there is no previous publication, it throws an `invalid` exception.
+
+Emits the [`afterRevertPublishedToPrevious` event](/docs/reference/server-events.md#afterrevertpublishedtoprevious) before returning, which includes the published document in its payload, allowing you to alter the returned published doc object.
```

Comment:
The payload of the event is an object, `{ draft }`, allowing reassignment of the value that will be returned by this method. Note that the draft has already been reverted in the database at this point, so if modification of the draft is intended an update call will also be needed.

Reply:
I got this description mostly from the `revertDraftToPublished` code comments. I don't really undestand why we'd suggest people use that payload object to alter the output at all since it doesn't actually change anything. I'm going to remove that.